skimage/filters/_frangi.py

```diff
@@ -0,0 +1,182 @@
+import numpy as np
+
+
+__all__ = ['frangi', 'hessian']
+
+
+def frangi(image, scale=(1, 10), scale_step=2, beta1=0.5, beta2=15,
+                  black_ridges=True):
+    """Returns image filtered with frangi filter.
+
+    Calculates the eigenvectors of the Hessian to compute the likeliness of
+    an image region to vessels, according to the method described in _[1].
+
+    Parameters
+    ----------
+    image : (N, M) ndarray
+        Array with input image data.
+    scale : tuple of floats, optional
+        The range of sigmas used, default [1 10].
+    scale_step : float, optional
+        Step size between sigmas, default 2.
```

Comment:
It is, we follow numpy's standard https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#docstring-standard